src/java/htsjdk/samtools/CRAMFileReader.java

```diff
@@ -56,32 +56,39 @@
     private ValidationStringency validationStringency;
 
     /**
-     * Open CRAM data for reading using either the file or the input stream
-     * supplied in the arguments. The
-     * {@link htsjdk.samtools.Defaults#REFERENCE_FASTA default} reference fasta
-     * file will be used.
+     * Create a CRAMFileReader from either a file or input stream using the {@link ReferenceSource#getDefaultCRAMReferenceSource() default}
+     * reference source.
      *
      * @param cramFile CRAM file to open
-     * @param inputStream   CRAM stream to read
+     * @param inputStream CRAM stream to read
+     *
+     * @throws IllegalArgumentException if the {@code cramFile} and the {@code inputStream} are both null
+     * @throws IllegalStateException if a {@link ReferenceSource#getDefaultCRAMReferenceSource() default}
+     * reference source cannot be acquired
      */
     public CRAMFileReader(final File cramFile, final InputStream inputStream) {
-        this(cramFile, inputStream, new ReferenceSource(Defaults.REFERENCE_FASTA));
+        this(cramFile, inputStream, ReferenceSource.getDefaultCRAMReferenceSource());
     }
 
     /**
-     * Open CRAM data for reading using either the file or the input stream
-     * supplied in the arguments.
+     * Create a CRAMFileReader from either a file or input stream using the supplied reference source.
      *
-     * @param cramFile            CRAM file to read
-     * @param inputStream              index file to be used for random access
+     * @param cramFile        CRAM file to read
+     * @param inputStream     index file to be used for random access
```

Comment:
`inputStream` is for the index file? The docs for the two-arg constructor `CRAMFileReader(final File cramFile, final InputStream inputStream)` that calls into this three-arg constructor imply that `inputStream` is connected to the cram file itself, not the index. Something is fishy here...